codebase/config/sync/islandora.settings.yml

```diff
@@ -1,5 +1,5 @@
 broker_url: 'tcp://activemq:61613'
-jwt_expiry: '+2 hour'
+jwt_expiry: '+168 hour'
```

Comment:
I think this looks fine, but I can't necessarily test it out. 
I do wonder if you want to say  `+7 days` instead?   I don't really care or have an opinion on that.